dockerfiles/cli/scripts/openshift/multi-user/deployPostgresOnly.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
```

Comment:
we're missing some utilities function like for creating something with oc create